rasa/core/actions/forms.py

```diff
@@ -550,13 +551,16 @@ async def _ask_for_slot(
     ) -> List[Event]:
         logger.debug(f"Request next slot '{slot_name}'")
 
-        action_to_ask_for_next_slot = action.action_from_name(
-            self._name_of_utterance(domain, slot_name), domain, self.action_endpoint
-        )
-        events_to_ask_for_next_slot = await action_to_ask_for_next_slot.run(
-            output_channel, nlg, tracker, domain
-        )
-        return events_to_ask_for_next_slot
+        action_to_ask_for_next_slot = self._name_of_utterance(domain, slot_name)
+
+        if action_to_ask_for_next_slot:
```

Comment:
this change was necessary to avoid running an non-existing action (it would fail silently before)